addons/web/static/src/views/fields/x2many/x2many_field.js

```diff
@@ -81,6 +81,9 @@ export class X2ManyField extends Component {
                 },
             });
         };
+
+        this.canOpenRecord = !(archInfo.editable || this.props.editable);
```

Comment:
this logic is only for lists right? When this is a kanban, we can always open the record? I'd maybe make that clear in this expression, like
```js
this.viewMode === "list" ? !(archInfo.editable || this.props.editable) : true;
```